src/utils/resource-utils.js

```diff
@@ -5,7 +5,7 @@ const extensionsGroups = [
   { type: RESOURCE_TYPE.pdf, extensions: ['pdf'] },
   { type: RESOURCE_TYPE.text, extensions: ['txt', 'doc', 'rtf', 'odt'] },
   { type: RESOURCE_TYPE.audio, extensions: ['aac', 'm4a', 'mp3', 'oga', 'ogg', 'wav', 'flac'] },
-  { type: RESOURCE_TYPE.image, extensions: ['jpg', 'jpeg', 'gif', 'png', 'tiff', 'raw', 'webp'] },
+  { type: RESOURCE_TYPE.image, extensions: ['jpg', 'jpeg', 'gif', 'png', 'tiff', 'raw', 'webp', 'svg'] },
```

Comment:
It would be beneficial to get rid of it, as it contributes a nice share to the bundle size (thanks to the `mime` package). The functionality of `mimeTypeHelper` I'd like keep though is to recognize the 20-30 most commonly used file extensions. I suggest we create a ticket to extract this into home-made code and then remove the package, whatcha think?

Reply:
Sounds good!